### PR TITLE
chore: Update importing-untrusted-tls-certificates.adoc

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

## What does this pull request change?
* Add cross-links between articles
* Update verification steps

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-3381

## Specify the version of the product this pull request applies to
`main` and cherry-pick to `7.52.x` and `7.53.x`

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by l0rd — APPROVED** [review]

File: modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ If the cluster is configured with an intermediate certificate, add the whole cha
 
 Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle`, and others. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded certificate. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
 
+See also: xref:deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc[].
```
Comment (themr0c) [inline modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc]: This line should go in the Additional resources section.

Comment (tolusha) [reply modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc]: Done

File: modules/administration-guide/pages/deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc [hunk 2]
```diff
@@ -54,13 +54,13 @@ spec:
 
 .Verification steps
 
-* Create and start a new workspace. Every container used by the workspace mounts a special volume that contains a file with the self-signed certificate. The repository's `.git/config` file contains information about the Git server host (its URL) and the path to the certificate in the `http` section (see Git documentation about link:https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslCAInfo[git-config]).
+* Create and start a new workspace. Every container used by the workspace mounts a special volume that contains a file with the self-signed certificate. The container's `/etc/gitconfig` file contains information about the Git server host (its URL) and the path to the certificate in the `http` section (see Git documentation about link:https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslCAInfo[git-config]).
 +
-.A `.git/config` file example
+.A `/etc/gitconfig` file example
```
Comment (max-cx) [inline modules/administration-guide/pages/deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc]:
```suggestion
.Contents of a `/etc/gitconfig` file
```

**Comment by max-cx** [comment]
@tolusha, could you please double-check if you still need `ca-certs` there:
image

**Comment by tolusha** [comment]
Updated

File: modules/administration-guide/pages/deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc [hunk 3]
```diff
@@ -54,13 +54,13 @@ spec:
 
 .Verification steps
 
-* Create and start a new workspace. Every container used by the workspace mounts a special volume that contains a file with the self-signed certificate. The repository's `.git/config` file contains information about the Git server host (its URL) and the path to the certificate in the `http` section (see Git documentation about link:https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslCAInfo[git-config]).
+* Create and start a new workspace. Every container used by the workspace mounts a special volume that contains a file with the self-signed certificate. The container's `/etc/gitconfig` file contains information about the Git server host (its URL) and the path to the certificate in the `http` section (see Git documentation about link:https://git-scm.com/docs/git-config#Documentation/git-config.txt-httpsslCAInfo[git-config]).
 +
-.A `.git/config` file example
+.Contents of a `/etc/gitconfig` file
```
Comment (max-cx) [inline modules/administration-guide/pages/deploying-che-with-support-for-git-repositories-with-self-signed-certificates.adoc]:
```suggestion
.Contents of an `/etc/gitconfig` file
```

**Review by max-cx — APPROVED** [review]

**Pull request merged**